### PR TITLE
Fix master ignition generation for DHCP of storage files do not exist

### PR DESCRIPTION
**Pull request by ori-amizur**
/cc @yevgeny-shnaidman 
/assign @yevgeny-shnaidman

### CODE REVIEW

File: render_files.py [hunk 1]
```diff
@@ -45,7 +45,8 @@ def add_dhcp_allocation_file(ignition_file, dhcp_allocation_file):
     try:
         with open(ignition_file, "r") as file_obj:
             data = json.load(file_obj)
-            storage_files = data['storage']['files']
+            data['storage'] = data.get('storage', dict())
+            storage_files = data['storage'].get('files')
```
Comment (yevgeny-shnaidman) [inline render_files.py:49]: file is supposed to be a list. Should not you return an empty list?

Comment (ori-amizur) [reply render_files.py:49]: It will behave the same if the storage_files is None at this point.  Look at the handling later

**Pull request merged**